### DISCUSSION TITLE
Error in Summary.factor

### DISCUSSION THREAD

**Discussion by MFZaini1984** [epiforecasts/EpiNow2#270] [Q&A]
Hi,

I'm trying to run EpiNow2 by using my own data. However, when I try to estimate, an error occurred as follows:
````
Error in Summary.factor(1:574, na.rm = TRUE) : 
  ‘max’ not meaningful for factors

**Comment by seabbs** [comment]
Thanks for opening this. 

Have you checked the examples as written work for you and that your data matches the types of the example data?

**Comment by MFZaini1984** [comment]
Dear Sir, Thank you for replying to my question. The problem had resolved yesterday as I changing the date format to "%Y-%m-%d". However, when I run the script to estimate the model, It took me more than 9 hours. Could you help me with this matter?
Herewith I attached the data and script that I use to estimate the [Rt.]
date | confirm
-- | --
24/1/2020 | 3
25/1/2020 | 1
26/1/2020 | 0
27/1/2020 | 2
28/1/2020 | 2
29/1/2020 | 0
30/1/2020 | 0
31/1/2020 | 0
1/2/2020 | 0
2/2/2020 | 0
3/2/2020 | 1
4/2/2020 | 3
5/2/2020 | 2
6/2/2020 | 0
7/2/2020 | 1
8/2/2020 | 1
9/2/2020 | 2
10/2/2020 | 0
11/2/2020 | 0
12/2/2020 | 1
13/2/2020 | 0
14/2/2020 | 2
15/2/2020 | 0
16/2/2020 | 1
17/2/2020 | 0
18/2/2020 | 0
19/2/2020 | 0
20/2/2020 | 0
21/2/2020 | 0
22/2/2020 | 0
23/2/2020 | 0
24/2/2020 | 0
25/2/2020 | 0
26/2/2020 | 0
27/2/2020 | 2
28/2/2020 | 1
29/2/2020 | 4
1/3/2020 | 0
2/3/2020 | 4
3/3/2020 | 3
4/3/2020 | 14
5/3/2020 | 5
6/3/2020 | 28
7/3/2020 | 10
8/3/2020 | 6
9/3/2020 | 19
10/3/2020 | 12
11/3/2020 | 19
12/3/2020 | 9
13/3/2020 | 42
14/3/2020 | 42
15/3/2020 | 186
16/3/2020 | 125
17/3/2020 | 120
18/3/2020 | 117
19/3/2020 | 110
20/3/2020 | 130
21/3/2020 | 153
22/3/2020 | 123
23/3/2020 | 212
24/3/2020 | 106
25/3/2020 | 172
26/3/2020 | 235
27/3/2020 | 130
28/3/2020 | 159
29/3/2020 | 150
30/3/2020 | 156
31/3/2020 | 140
1/4/2020 | 142
2/4/2020 | 208
3/4/2020 | 217
4/4/2020 | 150
5/4/2020 | 179
6/4/2020 | 131
7/4/2020 | 170
8/4/2020 | 156
9/4/2020 | 109
10/4/2020 | 118
11/4/2020 | 184
12/4/2020 | 153
13/4/2020 | 134
14/4/2020 | 170
15/4/2020 | 85
16/4/2020 | 110
17/4/2020 | 69
18/4/2020 | 54
19/4/2020 | 84
20/4/2020 | 36
21/4/2020 | 57
22/4/2020 | 50
23/4/2020 | 71
24/4/2020 | 88
25/4/2020 | 51
26/4/2020 | 38
27/4/2020 | 40
28/4/2020 | 31
29/4/2020 | 94
30/4/2020 | 57
1/5/2020 | 69
2/5/2020 | 105
3/5/2020 | 122
4/5/2020 | 55
5/5/2020 | 30
6/5/2020 | 45
7/5/2020 | 39
8/5/2020 | 68
9/5/2020 | 54
10/5/2020 | 67
11/5/2020 | 70
12/5/2020 | 16
13/5/2020 | 37
14/5/2020 | 40
15/5/2020 | 36
16/5/2020 | 17
17/5/2020 | 22
18/5/2020 | 47
19/5/2020 | 37
20/5/2020 | 31
21/5/2020 | 50
22/5/2020 | 78
23/5/2020 | 48
24/5/2020 | 60
25/5/2020 | 172
26/5/2020 | 187
27/5/2020 | 15
28/5/2020 | 10
29/5/2020 | 103
30/5/2020 | 30
31/5/2020 | 57
1/6/2020 | 38
2/6/2020 | 20
3/6/2020 | 93
4/6/2020 | 277
5/6/2020 | 19
6/6/2020 | 37
7/6/2020 | 19
8/6/2020 | 7
9/6/2020 | 7
10/6/2020 | 2
11/6/2020 | 31
12/6/2020 | 33
13/6/2020 | 43
14/6/2020 | 8
15/6/2020 | 41
16/6/2020 | 11
17/6/2020 | 10
18/6/2020 | 14
19/6/2020 | 6
20/6/2020 | 21
21/6/2020 | 16
22/6/2020 | 15
23/6/2020 | 3
24/6/2020 | 6
25/6/2020 | 4
26/6/2020 | 6
27/6/2020 | 10
28/6/2020 | 18
29/6/2020 | 3
30/6/2020 | 2
1/7/2020 | 1
2/7/2020 | 3
3/7/2020 | 5
4/7/2020 | 10
5/7/2020 | 5
6/7/2020 | 5
7/7/2020 | 6
8/7/2020 | 3
9/7/2020 | 6
10/7/2020 | 13
11/7/2020 | 8
12/7/2020 | 14
13/7/2020 | 7
14/7/2020 | 4
15/7/2020 | 5
16/7/2020 | 3
17/7/2020 | 18
18/7/2020 | 9
19/7/2020 | 15
20/7/2020 | 21
21/7/2020 | 15
22/7/2020 | 16
23/7/2020 | 9
24/7/2020 | 21
25/7/2020 | 23
26/7/2020 | 13
27/7/2020 | 7
28/7/2020 | 39
29/7/2020 | 13
30/7/2020 | 8
31/7/2020 | 12
1/8/2020 | 9
2/8/2020 | 14
3/8/2020 | 2
4/8/2020 | 1
5/8/2020 | 21
6/8/2020 | 15
7/8/2020 | 25
8/8/2020 | 7
9/8/2020 | 13
10/8/2020 | 11
11/8/2020 | 9
12/8/2020 | 11
13/8/2020 | 15
14/8/2020 | 20
15/8/2020 | 26
16/8/2020 | 25
17/8/2020 | 12
18/8/2020 | 7
19/8/2020 | 16
20/8/2020 | 5
21/8/2020 | 9
22/8/2020 | 8
23/8/2020 | 10
24/8/2020 | 7
25/8/2020 | 11
26/8/2020 | 6
27/8/2020 | 5
28/8/2020 | 10
29/8/2020 | 11
30/8/2020 | 17
31/8/2020 | 6
1/9/2020 | 14
2/9/2020 | 6
3/9/2020 | 14
4/9/2020 | 11
5/9/2020 | 6
6/9/2020 | 6
7/9/2020 | 62
8/9/2020 | 100
9/9/2020 | 24
10/9/2020 | 45
11/9/2020 | 182
12/9/2020 | 58
13/9/2020 | 47
14/9/2020 | 31
15/9/2020 | 23
16/9/2020 | 62
17/9/2020 | 21
18/9/2020 | 95
19/9/2020 | 20
20/9/2020 | 52
21/9/2020 | 57
22/9/2020 | 82
23/9/2020 | 147
24/9/2020 | 71
25/9/2020 | 111
26/9/2020 | 82
27/9/2020 | 150
28/9/2020 | 115
29/9/2020 | 101
30/9/2020 | 89
1/10/2020 | 260
2/10/2020 | 287
3/10/2020 | 317
4/10/2020 | 293
5/10/2020 | 432
6/10/2020 | 691
7/10/2020 | 489
8/10/2020 | 375
9/10/2020 | 354
10/10/2020 | 374
11/10/2020 | 561
12/10/2020 | 563
13/10/2020 | 660
14/10/2020 | 660
15/10/2020 | 589
16/10/2020 | 629
17/10/2020 | 869
18/10/2020 | 871
19/10/2020 | 865
20/10/2020 | 862
21/10/2020 | 732
22/10/2020 | 847
23/10/2020 | 710
24/10/2020 | 1228
25/10/2020 | 823
26/10/2020 | 1240
27/10/2020 | 835
28/10/2020 | 801
29/10/2020 | 649
30/10/2020 | 799
31/10/2020 | 659
1/11/2020 | 957
2/11/2020 | 834
3/11/2020 | 1054
4/11/2020 | 1032
5/11/2020 | 1009
6/11/2020 | 1755
7/11/2020 | 1168
8/11/2020 | 852
9/11/2020 | 972
10/11/2020 | 869
11/11/2020 | 822
12/11/2020 | 919
13/11/2020 | 1304
14/11/2020 | 1114
15/11/2020 | 1208
16/11/2020 | 1103
17/11/2020 | 1210
18/11/2020 | 660
19/11/2020 | 1290
20/11/2020 | 958
21/11/2020 | 1041
22/11/2020 | 1096
23/11/2020 | 1884
24/11/2020 | 2188
25/11/2020 | 970
26/11/2020 | 935
27/11/2020 | 1109
28/11/2020 | 1315
29/11/2020 | 1309
30/11/2020 | 1212
1/12/2020 | 1472
2/12/2020 | 851
3/12/2020 | 1075
4/12/2020 | 1141
5/12/2020 | 1123
6/12/2020 | 1335
7/12/2020 | 1600
8/12/2020 | 1012
9/12/2020 | 959
10/12/2020 | 2234
11/12/2020 | 1810
12/12/2020 | 1937
13/12/2020 | 1229
14/12/2020 | 1371
15/12/2020 | 1772
16/12/2020 | 1295
17/12/2020 | 1220
18/12/2020 | 1683
19/12/2020 | 1153
20/12/2020 | 1340
21/12/2020 | 2018
22/12/2020 | 2062
23/12/2020 | 1348
24/12/2020 | 1581
25/12/2020 | 1247
26/12/2020 | 2335
27/12/2020 | 1196
28/12/2020 | 1594
29/12/2020 | 1925
30/12/2020 | 1870
31/12/2020 | 2525
1/1/2021 | 2068
2/1/2021 | 2295
3/1/2021 | 1704
4/1/2021 | 1741
5/1/2021 | 2027
6/1/2021 | 2593
7/1/2021 | 3027
8/1/2021 | 2643
9/1/2021 | 2451
10/1/2021 | 2433
11/1/2021 | 2232
12/1/2021 | 3309
13/1/2021 | 2985
14/1/2021 | 3337
15/1/2021 | 3211
16/1/2021 | 4029
17/1/2021 | 3339
18/1/2021 | 3306
19/1/2021 | 3631
20/1/2021 | 4008
21/1/2021 | 3170
22/1/2021 | 3631
23/1/2021 | 4275
24/1/2021 | 3346
25/1/2021 | 3048
26/1/2021 | 3585
27/1/2021 | 3680
28/1/2021 | 4094
29/1/2021 | 5725
30/1/2021 | 5728
31/1/2021 | 5298
1/2/2021 | 4214
2/2/2021 | 3455
3/2/2021 | 4284
4/2/2021 | 4571
5/2/2021 | 3391
6/2/2021 | 3847
7/2/2021 | 3731
8/2/2021 | 3100
9/2/2021 | 2764
10/2/2021 | 3288
11/2/2021 | 3384
12/2/2021 | 3318
13/2/2021 | 3499
14/2/2021 | 2464
15/2/2021 | 2176
16/2/2021 | 2720
17/2/2021 | 2998
18/2/2021 | 2712
19/2/2021 | 2936
20/2/2021 | 2461
21/2/2021 | 3297
22/2/2021 | 2192
23/2/2021 | 2468
24/2/2021 | 3545
25/2/2021 | 1924
26/2/2021 | 2253
27/2/2021 | 2364
28/2/2021 | 2437
1/3/2021 | 1828
2/3/2021 | 1555
3/3/2021 | 1745
4/3/2021 | 2063
5/3/2021 | 2154
6/3/2021 | 1680
7/3/2021 | 1683
8/3/2021 | 1529
9/3/2021 | 1280
10/3/2021 | 1448
11/3/2021 | 1647
12/3/2021 | 1575
13/3/2021 | 1470
14/3/2021 | 1354
15/3/2021 | 1208
16/3/2021 | 1063
17/3/2021 | 1219
18/3/2021 | 1213
19/3/2021 | 1576
20/3/2021 | 1671
21/3/2021 | 1327
22/3/2021 | 1116
23/3/2021 | 1384
24/3/2021 | 1268
25/3/2021 | 1360
26/3/2021 | 1275
27/3/2021 | 1199
28/3/2021 | 1302
29/3/2021 | 941
30/3/2021 | 1133
31/3/2021 | 1482
1/4/2021 | 1178
2/4/2021 | 1294
3/4/2021 | 1638
4/4/2021 | 1349
5/4/2021 | 1070
6/4/2021 | 1300
7/4/2021 | 1139
8/4/2021 | 1285
9/4/2021 | 1854
10/4/2021 | 1510
11/4/2021 | 1739
12/4/2021 | 1317
13/4/2021 | 1767
14/4/2021 | 1889
15/4/2021 | 2148
16/4/2021 | 2551
17/4/2021 | 2331
18/4/2021 | 2195
19/4/2021 | 2078
20/4/2021 | 2341
21/4/2021 | 2340
22/4/2021 | 2875
23/4/2021 | 2847
24/4/2021 | 2717
25/4/2021 | 2690
26/4/2021 | 2776
27/4/2021 | 2733
28/4/2021 | 3142
29/4/2021 | 3332
30/4/2021 | 3788
1/5/2021 | 2881
2/5/2021 | 3418
3/5/2021 | 2500
4/5/2021 | 3120
5/5/2021 | 3744
6/5/2021 | 3551
7/5/2021 | 4498
8/5/2021 | 4519
9/5/2021 | 3733
10/5/2021 | 3807
11/5/2021 | 3973
12/5/2021 | 4765
13/5/2021 | 4855
14/5/2021 | 4113
15/5/2021 | 4140
16/5/2021 | 3780
17/5/2021 | 4446
18/5/2021 | 4865
19/5/2021 | 6075
20/5/2021 | 6806
21/5/2021 | 6493
22/5/2021 | 6320
23/5/2021 | 6976
24/5/2021 | 6509
25/5/2021 | 7289
26/5/2021 | 7478
27/5/2021 | 7857
28/5/2021 | 8290
29/5/2021 | 9020
30/5/2021 | 6999
31/5/2021 | 6824
1/6/2021 | 7105
2/6/2021 | 7703
3/6/2021 | 8209
4/6/2021 | 7748
5/6/2021 | 7452
6/6/2021 | 6241
7/6/2021 | 5271
8/6/2021 | 5566
9/6/2021 | 6239
10/6/2021 | 5671
11/6/2021 | 6849
12/6/2021 | 5793
13/6/2021 | 5304
14/6/2021 | 4949
15/6/2021 | 5419
16/6/2021 | 5150
17/6/2021 | 5738
18/6/2021 | 6440
19/6/2021 | 5911
20/6/2021 | 5293
21/6/2021 | 4611
22/6/2021 | 4743
23/6/2021 | 5244
24/6/2021 | 5841
25/6/2021 | 5812
26/6/2021 | 5803
27/6/2021 | 5586
28/6/2021 | 5218
29/6/2021 | 6437
30/6/2021 | 6276
1/7/2021 | 6988
2/7/2021 | 6982
3/7/2021 | 6658
4/7/2021 | 6045
5/7/2021 | 6387
6/7/2021 | 7654
7/7/2021 | 7097
8/7/2021 | 8868
9/7/2021 | 9180
10/7/2021 | 9353
11/7/2021 | 9105
12/7/2021 | 8574
13/7/2021 | 11079
14/7/2021 | 11618
15/7/2021 | 13215
16/7/2021 | 12541
17/7/2021 | 12528
18/7/2021 | 10710
19/7/2021 | 10972
20/7/2021 | 12366
21/7/2021 | 11985
22/7/2021 | 13034
23/7/2021 | 15573
24/7/2021 | 15902
25/7/2021 | 17045
26/7/2021 | 14516
27/7/2021 | 16117
28/7/2021 | 17405
29/7/2021 | 17170
30/7/2021 | 16840
31/7/2021 | 17786
1/8/2021 | 17150
2/8/2021 | 15764
3/8/2021 | 17105
4/8/2021 | 19819
5/8/2021 | 20596
6/8/2021 | 20889
7/8/2021 | 19257
8/8/2021 | 18688
9/8/2021 | 17236
10/8/2021 | 19991
11/8/2021 | 20780
12/8/2021 | 21668
13/8/2021 | 21468
14/8/2021 | 20670
15/8/2021 | 20546
16/8/2021 | 19740
17/8/2021 | 19631
18/8/2021 | 22242
19/8/2021 | 22948



</body>

</html>
`estimates <- epinow(reported_cases = reported_cases,
                    generation_time = generation_time,
                    delays = delay_opts(incubation_period, reporting_delay),
                    rt = rt_opts(prior = list(mean = 2, sd = 0.2)),
                    stan = stan_opts(cores = 4))`

**Comment by seabbs** [comment]
Are you interested in real-time or overall estimates? If overall only change the on argument in `rt_opts` to be on R0 and not R_t-1. That makes the model easier to fit but is not good for realtime performance.
